src/test/java/org/folio/rest/support/builders/RequestRequestBuilder.java

```diff
@@ -783,13 +803,30 @@ public RequestRequestBuilder withTags(Tags tags) {
       tags);
   }
 
-  private class ItemSummary {
+  public static class ItemSummary {
```

Comment:
Why is this class static, that suggests there can only ever be a single instance of this class?

Reply:
Because it is now possible to instantiate it in a test method and pass as a parameter to `RequestRequestBuilder#withItem` method.

Reply:
Does it need to be static to do that? Is that how nested classes work when they are made public?

Reply:
Non-static nested class can not be instantiated without an instance of outer class. Commonly, non-static classes used when they have to access to outer class's properties/methods/etc. An example can be `Itr` implementation in `ArrayList` when the `Itr` have to access to array of elements of the list. It also requires a bit complicated constructions: `new RequestRequestBuilder.new ItemSummary`.

Static nested classes does not use internal state of outer class, it is more like an additional namespace, and makes thinks a bit easier.

Reply:
Thanks for the explanation. I think I might have moved it to a separate class.